.gitignore

```diff
@@ -90,4 +90,7 @@ Fody/.DS_Store
 *.userprefs
 *.jtconfig
 
-*nsb_log*
\ No newline at end of file
+*nsb_log*
+
+samples/azure/shared-host/Version_5/tools/AzCopy/*
```

Comment:
Yes, we need it. 
Similar to `_zips` folder, we don't want anyone accidentally commit and push AzCopy files.